contracts/BSCValidatorSet.sol

```diff
@@ -74,7 +75,13 @@ contract BSCValidatorSet is IBSCValidatorSet, System, IParamSubscriber, IApplica
   uint256 public maxNumOfCandidates;
   uint256 public maxNumOfWorkingCandidates;
 
-  struct Validator{
+  // BEP-126 Fast Finality
+  uint256 public constant INIT_FINALITY_REWARD_RATIO = 10;
```

Comment:
Let us lower the ratio?

Reply:
how about like this?
epoch n system reward balance blance(n)
epoch n+1 system reward balance blance(n+1)
if 	blance(n+1)>blance(n)
	reward = INIT_FINALITY_REWARD_RATIO*(blance(n+1)-blance(n))
if reward =0
	reward = INIT_FINALITY_REWARD_RATIO*blance(n+1)